package.json

```diff
@@ -44,7 +44,7 @@
   },
   "dependencies": {
     "@abi-software/flatmap-viewer": "^2.6.2",
-    "@abi-software/sparc-annotation": "0.2.0",
+    "@abi-software/sparc-annotation": "0.3.0",
```

Comment:
The `session` seems to be set to `hhhhhhh` by default in `version 0.3.0`. Can you take a quick look?